components/FilesTable.tsx

```diff
@@ -8,44 +8,36 @@ const FilesTable = ({ files }) => {
   const columns = useMemo(
     () => [
       {
-        id: 'Local Id',
-        Header: 'Local id',
+        id: 'Id',
+        Header: 'id',
         accessor: (data) => String(data.id).padStart(9, '0'),
         Cell: ({ value }) => <span style={{ fontFamily: 'Mono', opacity: 0.4 }}>{value}</span>,
         disableFilters: true,
-        width: '7.8em',
+        width: '2.8em',
```

Comment:
this needs to stay wider for mobile responsiveness. can we keep it at the 7.8em? or iterate on something else that renders each column visibly while emulating the mobile devices in chrome's layout selector